.github/workflows/build.yml

```diff
@@ -23,7 +24,26 @@ jobs:
             build-net-tokio: true
             coverage: true
           - toolchain: 1.34.2
-    runs-on: ubuntu-latest
+        # Only run stable for MacOS and Windows. GitHub Actions syntax limitations
+        # lead to the unfortunate verboseness of the exclusions.
+        exclude:
+          - platform: macos-latest
```

Comment:
Is there not a way to capture this...slightly more compactly? Maybe in an include instead of an exclude?

Reply:
Heh I think I was tired.. fixed.